geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -133,8 +133,20 @@ pub struct ReplicaBlockInfo<'a> {
     pub block_height: Option<u64>,
 }
 
+/// Extending ReplicaBlockInfo by sending the transaction_entries_count.
+#[derive(Clone, Debug)]
+pub struct ReplicaBlockInfoV2<'a> {
+    pub slot: u64,
+    pub blockhash: &'a str,
+    pub rewards: &'a [Reward],
+    pub block_time: Option<UnixTimestamp>,
+    pub block_height: Option<u64>,
+    pub transaction_entries_count: u64,
```

Comment:
nit: rename all `transaction_entries_count` to just `transaction_count`

Reply:
Thanks Carl. Do we need to do the math to subtract the value of its parent bank?

Reply:
> Thanks Carl. Do we need to do the math to subtract the value of its parent bank?

Yes, until we add new fields to Bank, you definitely have to subtract the parent bank's value, as that field is cumulative since slot 0.